solr/core/src/test/org/apache/solr/handler/admin/PropertiesRequestHandlerTest.java

```diff
@@ -61,9 +64,11 @@ public void testDisabledRedaction() throws Exception {
 
   @SuppressWarnings({"unchecked"})
   private NamedList<Object> readProperties() throws Exception {
-    SolrQueryResponse rsp = new SolrQueryResponse();
-    SolrQueryRequest req = req();
-    h.getCoreContainer().getInfoHandler().getPropertiesHandler().handleRequestBody(req, rsp);
-    return (NamedList<Object>) rsp.getValues().get("system.properties");
+    SolrClient client = new EmbeddedSolrServer(h.getCore());
```

Comment:
Great!
And can you also use this approach for the other tests you added?  At least I think you added some that could use this technique; I'm not sure.